packages/client/hmi-client/src/types/workflow.ts

```diff
@@ -178,3 +178,16 @@ export interface AssetBlock<T> {
 	isCollapsed?: false;
 	asset: T;
 }
+
+export enum ChartSettingType {
+	VARIABLE = 'variable',
+	PARAMETER = 'parameter',
+	ERROR = 'error'
+}
```

Comment:
I sorta feel like we should introduce better naming now then deal with it later down the road.  Maybe:
- variable (eg simulate infected + recovered)
- variable-comparison (eg calibrate infected before and after)
- distribution-comparison (eg calibrate beta)
- error-distribution
- intervention

Reply:
Done